.projenrc.ts

```diff
@@ -1,19 +1,80 @@
 import { CdklabsConstructLibrary } from 'cdklabs-projen-project-types';
+import { DependencyType, JsonFile, JsonPatch } from 'projen';
+import { NpmAccess } from 'projen/lib/javascript';
 
 const project = new CdklabsConstructLibrary({
   author: 'AWS',
   authorAddress: 'aws-cdk-dev@amazon.com',
   cdkVersion: '2.80.0',
+  jsiiVersion: '~5.0.0',
   defaultReleaseBranch: 'main',
   name: '@aws-cdk/aws-lambda-dotnet',
   projenrcTs: true,
-  release: false,
+  npmAccess: NpmAccess.PUBLIC,
+  stability: 'experimental',
   repositoryUrl: 'https://github.com/cdklabs/awscdk-lambda-dotnet.git',
   autoApproveOptions: {
     allowedUsernames: ['aws-cdk-automation', 'dependabot[bot]', 'mergify[bot]'],
     secret: 'GITHUB_TOKEN',
   },
   autoApproveUpgrades: true,
+  prettier: true,
+  gitignore: [
+    '*.d.ts',
+    '*.generated.ts',
+    '*.js',
+    '**/integ.*.snapshot/asset.*/',
+    '**/bin',
+    '**/obj',
+    '/*.sln',
+  ],
+  publishToPypi: {
+    distName: 'cdklabs.aws-lambda-dotnet',
+    module: 'cdklabs.aws_lambda_dotnet',
+  },
+  publishToNuget: {
+    packageId: 'Cdklabs.AWS.Lambda.DotNet',
+    dotNetNamespace: 'Cdklabs.AWS.Lambda.DotNet',
+  },
+  publishToMaven: {
+    mavenGroupId: 'io.github.cdklabs',
+    javaPackage: 'io.github.cdklabs.aws.lambda.dotnet',
+    mavenArtifactId: 'cdklabs-aws-lambda-dotnet',
+    mavenEndpoint: 'https://s01.oss.sonatype.org',
+  },
+});
+
+project.package.addField('prettier', {
+  singleQuote: true,
+  semi: true,
+  trailingComma: 'es5',
 });
+project.eslint?.addRules({
+  'prettier/prettier': [
+    'error',
+    { singleQuote: true, semi: true, trailingComma: 'es5' },
+  ],
+});
+
+project.deps.addDependency(
+  '@aws-cdk/integ-tests-alpha@2.80.0-alpha.0',
+  DependencyType.TEST
+);
```

Comment:
Instead let's use `experimentalIntegRunner: true` as part of the `CdklabsConstructLibrary` options.
```suggestion
```

Reply:
It appears when I remove this, it sets the package version to latest. When the package version is latest, I get this error running the integration tests.

```Error: Package subpath './custom-resources/lib/helpers-internal' is not defined by "exports" in /Users/vinles/Labs/aws/awscdk-lambda-dotnet/node_modules/aws-cdk-lib/package.json
    at exportsNotFound (node:internal/modules/esm/resolve:294:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:641:9)
    at resolveExports (node:internal/modules/cjs/loader:591:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:668:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1130:27)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/Users/vinles/Labs/aws/awscdk-lambda-dotnet/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/vinles/Labs/aws/awscdk-lambda-dotnet/node_modules/@aws-cdk/integ-tests-alpha/lib/assertions/providers/provider.ts:4:1) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}```

Reply:
Updating `cdkVersion` fixes this as well, but that means that projects have to upgrade to that version in order to use the construct?

Reply:
Good question... could be that it was stated in another project.

Reply:
I know what the issue is, but don't have a way around it. 🤔 
happy to keep it like this for now.